### ISSUE TITLE
Suggestion change background color on hover

### ISSUE THREAD

**Issue by ifdiego** [blazkowolf/gruber-darker.nvim#11]
Hey,

I've been using this color, it's great, but I have a suggestion: change the background color on LSP hover to look like the same color as the vimdoc.

Example:
Here's how the default looks like in my neovim:
![one](https://github.com/blazkowolf/gruber-darker.nvim/assets/3338508/4a34f6f1-2fc8-4acf-a891-a85427271045)
Both backgrounds are the same color, which may be difficult to read.

And here's my suggestion:
![two](https://github.com/blazkowolf/gruber-darker.nvim/assets/3338508/4305830e-d9d1-433d-b07d-daf117370de3)

I think it looks better for reading with a darker background, but I'm not sure if could be changed here or just a personal modification in my own config.

It looks like I just need to change highlight NormalFloat to achieve that.
Maybe if it's ok, I could make a pull request.

**Comment by blazkowolf** [comment]
Thanks for your interest! I like the idea of changing this to a more legible color - you're more than welcome to make a PR. One thing I would suggest though is not a darker color, but a slightly lighter color than what is usually the editor background color. This would be a bit more visually consistent. Darker colors (at least in some other themes I've seen) are used for non-active buffers.

My apologies to you and other users of the color scheme for lack of updates. I'm currently tied up in other interests. I don't plan on _never_ making changes again, but updates will be sparse.

**Comment by blazkowolf** [comment]
Regarding your query - One thing I will say is this is not an issue with borders on popups. That's why I never ran into this, but it totally makes sense to change the color regardless

**Comment by ifdiego** [comment]
Indeed a lighter background looks way better! Previously I changed to the [bg-1] color in the palette, now I tried with [bg+1] and it looks like this:
![lightbg](https://github.com/blazkowolf/gruber-darker.nvim/assets/3338508/20e0a4de-b6ad-40dc-886b-c5d83ad4d270)

I already submit the PR with this change!

I have only changed the NormalFloat highlight to use [bg+1] background.
Changing the `Float`, `FloatBorder`, `FloatShadow`, `FloatShadowThrough` and `FloatTitle` highlights didn't make big differences, so I kept the defaut.

I also tested `nvim-cmp` with borders and `telescope` to see if/how would affect, but everything looks like usual.

Thanks!
